### ISSUE TITLE
Wrong bx-wrapper bottom margin with controls=true and pager=false

### ISSUE THREAD

**Issue by Vincent87** [stevenwanderski/bxslider-4#647]
A small bug at line 217. You should not remove the bottom margin when controls are used (without pager) and thus add  && !slider.settings.controls in your test :

// make modification to the wrapper (.bx-wrapper)
if(!slider.settings.pager && !slider.settings.controls) {
   slider.viewport.parent().css({
   margin: '0 auto 0px'
   });
}

**Comment by Tidal-Wave** [comment]
Fixed in 4.2.2

**Comment by coachwei** [comment]
This fix is problematic. It creates the wrong margin-bottom when "pager" is "false" and there aren't enough slides. Try the followings:

HTML is:

```
<div class="slider7">
  <div class="slide"><img src="http://placehold.it/200x150&text=FooBar1"></div>
  <div class="slide"><img src="http://placehold.it/200x150&text=FooBar2"></div>
</div>
```

and JS is:
```
$(document).ready(function(){
  $('.slider7').bxSlider({
    slideWidth: 200,
    minSlides: 4,
    maxSlides: 5,
    slideMargin: 10,
   pager: false
  });
});
```
